doc/README.md

```diff
@@ -160,11 +161,6 @@ below.
   </tbody>
 </table>
 
-## Available options
-
-Check out [this page](common_info.md) for more information on available options
-that these rules honor during a build.
-
```

Comment:
I felt that this was hidden behind the other rule docs table and was hard to find

Reply:
We could land that as its own PR.